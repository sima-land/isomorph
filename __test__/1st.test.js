import React from 'react';
import { shallow } from 'enzyme';
import TestComponent from './index';

describe('test', () => {
  it('test1', () => {
    const wrapper = shallow(<TestComponent />);
    expect(wrapper).toMatchSnapshot();
  });

  it('test2', () => {
    const wrapper = shallow(<TestComponent />);
    expect(wrapper.find('h1')).toHaveLength(1);
  });
});
