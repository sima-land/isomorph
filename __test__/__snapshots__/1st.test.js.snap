// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test test1 1`] = `
<div
  className="App"
>
  <h1>
    Hello Listar
  </h1>
</div>
`;
